vectordatasource/transform.py

```diff
@@ -9403,7 +9407,15 @@ def cut(self, shape, props, fid):
                 if not cut_shape.is_empty:
                     new_props = props.copy()
                     for disputant in non_claim_disputants:
-                        new_props['kind:' + disputant] = unrecognized
+                        new_props['kind:' + disputant] = 'unrecognized_disputed_reference_line'
+
+                    for recognizant in recognizants:
+                        new_props['kind:' + recognizant] = 'country'
+
+                    for claimant in claimants:
+                        new_props['kind:' + claimant] = 'country'
+
+                    new_props['kind'] = 'disputed_reference_line'
```

Comment:
Modulo other PR comments and IRL discussion update `disputed_reference_line` to `disputed`?